CMakeLists.txt

```diff
@@ -15,58 +15,99 @@ option(SPNG_STATIC "Build static lib" ON)
 option(BUILD_EXAMPLES "Build examples" ON)
 
 include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
 
 # Allow the usage of [PackageName]_ROOT variables for FindPackage
-if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12")
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
```

Comment:
You cannot use `VERSION_GREATER_EQUAL`, since it has been introduced in CMake 3.7 and this project targets CMake 3.0. I know you haven't added `VERSION_GREATER_EQUAL` yourself (it was added in commit 82b0c5107d3ca3bf9386496c5c698f025df92858), but it'd be still nice to fix it.

You could either:

- Bump the cmake_minimum_required version to 3.7 - shouldn't cause issues to anyone, since CMake 3.7 is six years old and this script required it already anyway
- Change the condition to `NOT CMAKE_VERSION VERSION_LESS "3.12"`